src/workerd/io/io-context.h

```diff
@@ -233,8 +233,8 @@ class IoContext final: public kj::Refcounted, private kj::TaskSet::ErrorHandler
 
   using IncomingRequest = IoContext_IncomingRequest;
 
-  const Worker& getWorker() {
-    return *worker;
+  kj::Arc<Worker>& getWorker() {
```

Comment:
this getWorker() is often used as a temporory to fetch compat flags or script id. It makes sense to return a reference (which simple `auto` won't accept).